terraform/charm/large_deployment/main.tf

```diff
@@ -0,0 +1,185 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+locals {
+  apps = [
+    for app in concat(var.apps != null ? var.apps : []) : app if app != null
+  ]
+
+  apps_in_main_model = [
```

Comment:
I guess this is not really used? Can we remove it?

Reply:
Good catch